models/prices/base/prices_base_tokens.sql

```diff
@@ -28,5 +28,13 @@ FROM
     ('based-basedmarkets','base','BASED',0xba5e6fa2f33f3955f0cef50c63dcc84861eab663,18),
     ('usdc-usd-coin', 'base', 'USDC', 0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913, 6),
     ('wsteth-wrapped-liquid-staked-ether-20', 'base', 'wstETH',0xc1CBa3fCea344f92D9239c08C0568f6F2F0ee452,18),
-    ('aero-aerodrome-finance', 'base', 'AERO', 0x940181a94A35A4569E4529A3CDfB74e38FD98631, 18)
+    ('aero-aerodrome-finance', 'base', 'AERO', 0x940181a94A35A4569E4529A3CDfB74e38FD98631, 18),
+    ('degen-degen-base', 'base', 'DEGEN', 0x4ed4e862860bed51a9570b96d89af5e1b0efefed,18),
+    ('ftw-friendtech33', 'base', 'FTW', 0x3347453ced85bd288d783d85cdec9b01ab90f9d8,9),
```

Comment:
This token is not active:
https://api.coinpaprika.com/v1/coins/ftw-friendtech33

We may need to remove `FTW` from this file.
```suggestion
```